DATA/production/qc-async/trd.json

```
@@ -23,7 +23,7 @@
       }
     },
     "tasks": {
-      "TRDDigits": {
+      "Digits": {
```

```suggestion
      "TRDDigits": {
```
This was like this for a reason - to avoid key collisions in merged workflows. Since we combine JSONs for async processing, this will be supressed or will suppress a different task with the name `Digits`.

If you need more reading: https://github.com/AliceO2Group/QualityControl/blob/master/doc/Advanced.md#merging-multiple-configuration-files-into-one

---

Yip I forgot about whether the tasks should have the detector in them, and then seem to have remembered the wrong way around.   fixing.